### PR TITLE
[test] Track gzipped file size in metadce code size tests. NFC

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

**Comment by sbc100** [comment]
> What do you see as the benefit here?

When I make changes that I hope will save on code size if you would good to know if they have real world benefits, since I believe most folks in the real world are using compression on the wire, right?

**Comment by kripken** [comment]
True. But usually the two sizes are correlated. I guess the recent embind codegen is a good example where there might be a divergence though (bigger size but repeating and compressible code patterns). So it makes sense to me.

File: test/test_other.py [hunk 1]
```diff
@@ -8218,11 +8227,14 @@ def run_metadce_test(self, filename, args=[], expected_exists=[], expected_not_e
       wasm_size = os.path.getsize('a.out.nodebug.wasm')
       size_file = expected_basename + '.size'
       js_size = os.path.getsize('a.out.js')
+      gz_size = get_file_gzipped_size('a.out.js')
```
Comment (kripken) [inline test/test_other.py:8230]: Why measure gzip of the JS and not the wasm?

**Comment by sbc100** [comment]
I'm happy to have gzip size for wasm too, but the PR that motivated me to create this was https://github.com/emscripten-core/emscripten/pull/20877.   Here you can see a clear cosesize win, but if you look at gzip sizes then its less clear.

**Pull request merged**